### DISCUSSION TITLE
Zigbee support

### DISCUSSION THREAD

**Discussion by jonczykp** [pvvx/THB2#17] [Ideas]
Are you planning to support Zigbee for th05 or other devices supported here?

**Comment by pvvx** [comment]
The datasheet about PHY6222 says that it does not support Zigbee.
PHY6225/6226 support Zigbee.

**Comment by papagenic** [comment]
reviving this discussion. 
Trying to look at tuya-ts0001-power (https://www.zigbee2mqtt.io/devices/TS0001_power.html#tuya-ts0001_power)
They are zigbee devices based on the chip PHY6222QCW04I  which seems to support zigbee . Anybody knows if there is a zigbee SDK for this chip?

**Comment by pvvx** [comment]
[Tuya TS0001_TZ3000](https://pvvx.github.io/TS0001_TZ3000/)
Module ZTU - TLSR8258
TLSR825x - Zigbee info: IEEE: **a4:c1:38**:xx:xx:xx:xx:xx

**Comment by papagenic** [comment]
Well, the TS001_TZ3000 modules currently sold do not ship with a ZTU module, but rather with a 6222BTS module which seems to be a pin to pin replacement of the ZTU module  based upon the PHY6222QCW04I  chip . I am looking to replace the firmware because the routing functionality does not work properly.
https://www.alibaba.com/product-detail/PHY6222-chip-factory-Wireless-Transceiver-Serial-1600692867172.html

**Comment by pvvx** [comment]
> 6222BTS

Tuya does not have such modules:  https://solution.tuya.com/hardware

https://solution.tuya.com/hardware?chipPlatformCode=PHY6222

> I am looking to replace the firmware because the routing functionality does not work properly.

Nobody is stopping you from replacing the firmware.
The programmer has been around for a long time: https://github.com/pvvx/THB2/blob/master/rdwr_phy62x2.py

At the moment, I and [froloffw7](https://github.com/froloffw7) have only translated (reverse engineering) the BLE SDK to GCC with all source codes.

**Comment by papagenic** [comment]
There are really 2 questions:
1- how can tuya devices like this: https://www.aliexpress.com/item/1005007578256078.html be based upon a TI6222BTS module? is the TI6666BTS  a perfect clone of the ZTU? Other possibilities?
2-  I understand there are tools to flash a firmware to this device, but if we want to create a new firmware we need a zigbee sdk for the py62x2. Is there such thing available?

**Reply by pvvx** [reply]
https://pvvx.github.io/TS0001_TZ3000_fdxihpp7/
